services/client/src/components/event/time-to-event.js

```diff
@@ -13,11 +13,11 @@ const TimeToEvent = ({ startTime }) => {
         span {
           color: #989fa5;
           font-style: italic;
-          font-weight: 200;
+          font-weight: 500;
         }
       `}</style>
       <span className="humanize">
-        ( {duration > 0 ? "next" : "last"} event {duration.humanize(true)} )
+        - {duration > 0 ? "next" : "last"} event {duration.humanize(true)}
```

Comment:
You may need to add a `&nbsp;` here to provide the space character in front of the `-`, or you could also try making the `-` a string literal so React knows the keep the space in front:
```js
<span className="humanize">
  {' - '}{duration > 0 ? "next" : "last"} event {duration.humanize(true)}
</span>
```